src/main/java/sirius/web/data/ExcelExport.java

```diff
@@ -491,4 +508,34 @@ private boolean isNumeric(Object data) {
 
         return (data instanceof Amount) && ((Amount) data).isFilled();
     }
+
+    public String getMaxRowsReachedMessage() {
```

Comment:
does this need to be public? also, this isn't a real getter - determineMaxRowsMessage or the like?